docs/README.md

```diff
@@ -375,8 +375,8 @@ Example:
 ```text
 set_global hv_dom0_mem_settings "dom0_mem=800M,max:800M"
 set_global hv_dom0_cpu_settings "dom0_max_vcpus=1 dom0_vcpus_pin"
-set_global hv_eve_mem_settings "eve_mem=500M,max:500M"
+set_global hv_eve_mem_settings "eve_mem=6500M,max:6500M"
```

Comment:
Did you intentionally want to have this example be 13x larger or did you intend to add 150M as for the example below?

Reply:
Oh, typo, fixed